src/test/java/com/znsio/sample/e2e/screen/android/amazonsearch/AmazonProductScreenAndroid.java

```diff
@@ -0,0 +1,55 @@
+package com.znsio.sample.e2e.screen.android.amazonsearch;
+
+import com.znsio.e2e.runner.Runner;
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.entities.SAMPLE_TEST_CONTEXT;
+import com.znsio.sample.e2e.screen.amazonsearch.AmazonProductScreen;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+
+public class AmazonProductScreenAndroid extends AmazonProductScreen {
+    private final Driver driver;
+    private final Visual visually;
+    private static final String SCREEN_NAME = AmazonProductScreenAndroid.class.getSimpleName();
+    private static final Logger LOGGER = Logger.getLogger(SCREEN_NAME);
+    private static final By byProductTitle = By.xpath("//android.widget.TextView[contains(@text, '" + SAMPLE_TEST_CONTEXT.PRODUCT_NAME + "')]");
+    private static final By byAddToCartButton = By.xpath("//android.widget.Button[@text='Add to Cart']");
+
+    public AmazonProductScreenAndroid(Driver driver, Visual visually) {
+        this.driver = driver;
+        this.visually = visually;
+        visually.checkWindow(SCREEN_NAME, "Home Page");
+    }
+
+    @Override
+    public boolean isProductDetailsDisplayed() {
+        LOGGER.info("Verifying product details like Product Title");
+        driver.waitTillElementIsPresent(byProductTitle);
+        boolean isProductTitle = driver.isElementPresent(byProductTitle);
```

Comment:
no need to add an additional variable isProductTiltle

Reply:
It was suggested not use driver.iselementPresent() as a return statement, @vishal07111998  if it is okay then i can chnage it.